src/themes/teams/components/Icon/iconStyles.ts

```diff
@@ -14,6 +14,8 @@ const sizes = new Map([
   ['small', 14],
   ['normal', 16],
   ['large', 20],
+  ['x-large', 24],
```

Comment:
lets move it to icon's variables, specifically:

```jsx
<Icon size='large' variables={{ sizeModifier: 'xx' }}>
```

And adjust the logic of mapping `size` prop + `sizeModifier` variable => size value.

With that we will remove the following problems:
- we won't introduce any changes to the `Icon's` API - note, currently `xx`-prefixed values are not supported, according to the Icon's typings (and those are used for docs generation)
- won't introduce inconsistency in type for `size` prop - note, currently `xx` prefixes are only supported for large values, and for other ones `size` type was left unchanged.

I am sure this problem (with extendable enums) will be addressed in future by Stardust, but for now using variables seems to be less evil we can do to circumvent the problem.

Reply:
@kuzhelov The better approach will be to let the user enter a number as a size. Something as sizeInNumber

Reply:
Done